nslsii/common/ipynb/logutils.py

```diff
@@ -34,6 +35,9 @@ def log_exception(ipyshell, etype, evalue, tb, tb_offset=None):
     tb_lines = traceback.format_exception(etype, evalue, tb)
     for tb_line in tb_lines:
         ipyshell.logger.log_write(tb_line, kind="output")
-    ipyshell.showtraceback((etype, evalue, tb), tb_offset=tb_offset)
+    ipyshell.showtraceback((etype, evalue, tb), tb_offset=tb_offset, exception_only=True)
+
+    logging.getLogger("bluesky.ipython").exception(evalue)
```

Comment:
If this is scoped to the `'bluesky'` logger it will be affected by that logger's level threshold. I think it might make sense to put this in a separate logger namespace, maybe `'nslsii.ipython'`.

Reply:
I agree.